packages/@studiocms/cloudinary-image-service/package.json

```
@@ -54,7 +55,8 @@
     "astro-integration-kit": "catalog:"
   },
   "devDependencies": {
-    "@types/node": "catalog:"
+    "@types/node": "catalog:",
+    "vitest": "catalog:"
```

```suggestion
    "@types/node": "catalog:"
```

again? :sweat_smile: this is handled from workspace